public/app/core/utils/explore.ts

```diff
@@ -52,7 +55,13 @@ export interface GetExploreUrlArguments {
 }
 
 export function generateExploreId() {
-  return nanoid(3);
+  while (true) {
+    const id = nanoid(3);
+
+    if (!/^\d+$/.test(id)) {
```

Comment:
Why do we need it? 🤔

Reply:
yes, numeric-only panel ids will lose the ordering when parsed, this prevents that